test/performance-test/amp-caching-module-test.js

```diff
@@ -66,50 +66,50 @@ test.after(() => {
   server.close();
 });
 
-test('should enable fast script response when the browser cache fails to respond', async t => {
-  let browser = await puppeteer.launch();
-  let page = await browser.newPage();
+async function performanceOfScripts(useServiceWorker) {
+  const browser = await puppeteer.launch();
+  const page = await browser.newPage();
   await interceptAmpScripts(page);
-  // load page for the first time
-  await page.goto('http://localhost:6881/test/accordian.amp.html', {
-    waitUntil: 'networkidle0',
-  });
-  // let cache expire
-  await sleep(4000);
-  // reload the page
-  await page.reload({
-    waitUntil: 'networkidle0',
-  });
-  const totalAmpScriptsLoadTimeWithoutSW = await page.evaluate(() => {
+
+  try {
```

Comment:
is this just needed as a overall catch all?

Reply:
Yes, the upgraded version of `ava` will timeout more aggressively and without a catch the error isn't reported thru.